### ISSUE TITLE
Just wondering the process 

### ISSUE THREAD

**Issue by DarylRob** [berton7/kek-monitors#2]
Hi, Back again still trying to learn. 

Just wondering what's the actually process in an ELI5 format. 

So the scrapers scrape the website and add to the database?
when the scraper adds to the database and there is a change (? new product or change to existing?) then it sends out a discord notifcation and updates the database?

Is that it in simplistic terms?

Thanks

**Comment by berton7** [comment]
Exactly what you said, plus the shoes get sent to the monitor if available.

To avoid opening issues, if you have more questions, please join the new Discord server: https://discord.gg/76r8GJyeZZ

**Comment by DarylRob** [comment]
Sorry mate for opening issues, cant find the chat channels or questions channel in the discord
